online-shop/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
Just a general remark (and will apply to all .spec): no need to add them if you don't want to test the component. Also the same apply for scss (of course, if they are empty). + If html files only have few lines of code and no fancy logic, you can put the html code inside your .ts file.

Reply:
Okay.